faqs/galaxy/histories_transfer_entire_histories.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Transfer entire histories
+area: histories 
+box_type: tip
+layout: faq
+contributors: [AnomalyCodes]
+---
+To transfer an entire history from one galaxy instance to another, follow these steps:
+
+1. Click on {% icon galaxy-gear %} in the history panel of the *sender* galaxy instance
+2. Click on **Export to File**
+3. Select either exporting history **to a link** or **to a remote file**
+4. Click on the link text to generate a new archive for the history *if* exporting to a link
+5. Wait for the link to generate
+6. Copy the link address or click on the generated link to download the history archive
+7. Click on **User** on the top menu of the *receiver* galaxy instance
```

Comment:
```suggestion
7. Click on **User** on the top menu of the *receiver* Galaxy instance
```

Reply:
Thank you, @beatrizserrano for this suggestion!